app/views/about-your-organisation/index.html

```diff
@@ -38,81 +94,149 @@ <h1 class="govuk-heading-xl">
 
   <div class="govuk-grid-row govuk-!-margin-top-3">
     <div class="govuk-grid-column-two-thirds">
-      <h3 class="govuk-heading-m govuk-!-font-size-27 govuk-!-margin-top-0">
-        <a href="/about-your-organisation/contact">Contact details</a>
-      </h3>
-
-      {% set contactAddressHtml %}
-        {% if data["building-and-street"] or data["building-and-street-2"] or data["town-or-city"] or data["county"] or data["postcode"] %}
-          {% if data["building-and-street"] %}
-            {{ data["building-and-street"] }}<br />
-          {% endif %}
-          {% if data["building-and-street-2"] %}
-            {{ data["building-and-street-2"] }}<br />
-          {% endif %}
-          {% if data["town-or-city"] %}
-            {{ data["town-or-city"] }}<br />
-          {% endif %}
-          {% if data["county"] %}
-            {{ data["county"] }}<br />
-          {% endif %}
-          {% if data["postcode"] %}
-            {{ data["postcode"] }}<br />
-          {% endif %}
-        {% else %}
-          Empty
-        {% endif %}
-      {% endset %}
-
+      <h3 class="govuk-heading-m">Contact details</h3>
       {{ govukSummaryList({
         rows: [{
+          classes: macros.requiredSummaryRowClasses("email"),
           key: {
             text: "Email address"
           },
           value: {
-            text: value("email") or "Empty",
-            classes: "app-summary-list__value--empty" if not value("email")
+            text: value("email") or "Empty"
+          },
+          actions: {
+            items: [{
+              href: "/about-your-organisation/contact",
+              text: "Change",
+              visuallyHiddenText: "email address"
+            }]
           }
         }, {
+          classes: macros.requiredSummaryRowClasses("telephone"),
           key: {
             text: "Telephone number"
           },
           value: {
-            text: value("telephone") or "Empty",
-            classes: "app-summary-list__value--empty" if not value("telephone")
+            text: value("telephone") or "Empty"
+          },
+          actions: {
+            items: [{
+              href: "/about-your-organisation/contact",
```

Comment:
Missing an anchor? Or is better not to have these? 🤔 

```suggestion
              href: "/about-your-organisation/contact#telephone",
```

Reply:
Given these would link to the input not the label, seems problematic. Ideally we would add `id`s to the labels, but will leave for now.